packages/pds/src/content-reporter/index.ts

```diff
@@ -42,12 +42,13 @@ export class ContentReporter {
 
   checkRecord(opts: { record: RepoRecord; uri: AtUri; cid: CID }) {
     const { record, uri, cid } = opts
-    let content = uri.rkey
+    let content = ''
     if (isProfile(record)) {
```

Comment:
we *could* add profile description here, though that could create too much noise. I think this is a good place to start and see how this goes, don't want to swap the mod queue.